src/js/edu-benefits/1990e/containers/ConfirmationPage.jsx

```diff
@@ -31,8 +36,79 @@ class ConfirmationPage extends React.Component {
   }
 
   render() {
+    const form = this.props.form;
+    const response = this.props.form.submission.response
+      ? this.props.form.submission.response.attributes
+      : {};
+    const name = form.applicantInformation.data.relativeFullName;
+    const benefit = form.benefitEligibility.data.benefit;
+
+    const docExplanation = this.state.isExpanded
+      ? (<div className="usa-accordion-content">
+        <p>In the future, you might need:</p>
+        <ul>
+          <li>Your reserve kicker</li>
+          <li>Documentation of additional contributions that would increase your monthly benefits.</li>
+        </ul>
+        <p>Documents can be uploaded using the <a href="https://gibill.custhelp.com/app/utils/login_form/redirect/account%252">GI Bill site</a>.</p>
+      </div>)
+      : null;
+
     return (
       <div className="edu-benefits-submit-success">
+        <h3 className="edu-page-title">Claim received</h3>
+        <p>Normally processed within <strong>30 days</strong></p>
+        <p>
+          We may contact you for more information or documents.<br/>
+          <i>Please print this page for your records.</i>
+        </p>
+        <div className="inset">
+          <h4>Education Benefit Claim <span className="additional">(Form 22-1990e)</span></h4>
+          <span>for {name.first} {name.middle} {name.last} {name.suffix}</span>
+
+          <ul className="claim-list">
+            <li>
+              <strong>Selected benefit</strong><br/>
+              {benefitsLabels[benefit]}
+            </li>
+            <li>
+              <strong>Confirmation number</strong><br/>
+              <span>{response.confirmationNumber}</span>
```

Comment:
Yeah, that's the number in dev/staging, it should be a more appropriate looking string in prod. Though I think Alex was opening an issue about making sure that is the case.